src/modules/data.js

```diff
@@ -0,0 +1,94 @@
+/* eslint-disable no-underscore-dangle */
+import heartIcon from '../assets/img/heart-icon.png';
+import { fetchLikes, like } from './movie-likes.js';
+import { movieUrl } from './from-api.js';
+import handleCommentPopup from './commentsPopup.js';
+import handleReservePopup from './reservePopup.js';
+import movieCounter from './movie-counter.js';
+
+const moviesList = document.querySelector('.movies-list');
+
+const likesCheck = (movieId, node) => {
+  fetchLikes().then((data) => {
+    try {
+      const movieLikes = data.filter((movie) => movie.item_id === movieId);
+      if (typeof movieLikes[0] === 'undefined') {
+        node.innerText = `${0} Likes`;
+      } else {
+        node.innerText = `${movieLikes[0].likes} Likes`;
+      }
+    } catch {
+      // throw new Error();
```

Comment:
- [ ] Kindly remove unwanted codes completely instead of adding them as comments. It makes your code look kind of messy and prone to bugs :wink: